lti_tool/models.py

```diff
@@ -466,6 +466,35 @@ def __str__(self):
         return self.title if self.title else self.id_on_platform
 
 
+class LtiLineItem(models.Model):
+    """Represents an LTI LineItem for assignment scoring.
+
+    The line item scope and methods are described in the LTI Assignment and Grade
+    Services 2.0 spec at
+    https://www.imsglobal.org/spec/lti-ags/v2p0#line-item-service-scope-and-allowed-http-methods
+    """
+
+    context = models.ForeignKey(
+        LtiContext,
+        related_name="line_items",
+        on_delete=models.CASCADE,
+        verbose_name=_("context"),
+    )
+    url = models.URLField(_("URL on Platform"))
+    maximum_score = models.FloatField(_("Maximum score"))
+    label = models.CharField(_("Label"), max_length=255)
+    tag = models.CharField(_("Tag"), max_length=255)
+    resource_id = models.CharField(_("Resource ID"), max_length=255)
```

Comment:
Do you have any issue with adding `blank=True` to these, in case this eventually gets exposed in a form or something?

Reply:
I don't. That's a fair request

- [ ] Add `blank=True` to fields that are allowed to be empty

Reply:
I specifically avoided using `blank` on `label`, because [the spec says](https://www.imsglobal.org/spec/lti-ags/v2p0/#label): "The label is a short string with a human readable text for the line item. It MUST be specified and not blank when posted by the tool. A platform must always include the label." Since this model is built to reflect what is on the platform (indeed, we can't include an object until it has an `id` from the platform, the spec suggests we will have a label, regardless of whether a tool provided one.

Reply:
Yep, we're on the same page regarding `label`.